.github/infrastructure/conformance/azure/conf-test-azure-cosmosdb.bicep

```diff
@@ -51,6 +51,10 @@ resource cosmosDb 'Microsoft.DocumentDB/databaseAccounts@2021-04-15' = {
               '/partitionKey' // Defined by conformance test state.go
             ]
           }
+          // Based on https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/time-to-live,
+          // if defaultTtl is not set, the item will never expire and also ttl is not enabled at all so
+          // ttl set on a per item basis will not be honored.
+          defaultTtl: 86400 // 1 day
```

Comment:
Can you set this to `-1`? per the docs, it will enable the TTLs but without a default TTL

Reply:
In test infra why should we not delete data that's more than 1 day old?

Reply:
I think we referred: https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/time-to-live#time-to-live-for-containers-and-items
Just wanted to understand: why didn't we go ahead with expiring test data by-default, let's say after 1 day?